packages/plugin-data/src/commands/force/data/soql/query.ts

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// This is the legacy converted command file. Ignoring code-coverage since this is generated.
+// THIS SHOULD BE REMOVED WHEN CONVERTED TO EXTEND SfdxCommand
+/* istanbul ignore file */
+
+import * as os from 'os';
+import { flags, FlagsConfig, SfdxCommand, SfdxResult } from '@salesforce/command';
+import { Connection, Messages, Org } from '@salesforce/core';
+import { asPlainObject, toJsonMap } from '@salesforce/ts-types';
+import { Tooling } from '@salesforce/core/lib/connection';
+import { CsvReporter, FormatTypes, HumanReporter, JsonReporter } from '../../../../api/data/soql/reporters';
+import { DataSoqlQueryResult } from '../../../../api/data/soql/dataSoqlQueryTypes';
+import { SoqlQuery, SoqlQueryResult } from '../../../../api/data/soql/soqlQuery';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-data', 'soql.query');
+
+export class DataSoqlQueryCommand extends SfdxCommand {
+  public static readonly description = messages.getMessage('description');
+  public static readonly longDescription = messages.getMessage('longDescription');
+  public static readonly requiresProject = false;
+  public static readonly requiresUsername = true;
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+
+  public static readonly flagsConfig: FlagsConfig = {
+    query: flags.string({
+      char: 'q',
+      required: true,
+      hidden: false,
+      description: messages.getMessage('queryToExecute'),
+      longDescription: messages.getMessage('queryLongDescription'),
+    }),
+    usetoolingapi: flags.boolean({
+      char: 't',
+      required: false,
+      hidden: false,
+      default: false,
+      description: messages.getMessage('queryToolingDescription'),
+      longDescription: messages.getMessage('queryToolingLongDescription'),
+    }),
+    resultformat: flags.enum({
+      char: 'r',
+      description: messages.getMessage('resultFormatDescription'),
+      longDescription: messages.getMessage('resultFormatLongDescription'),
+      options: ['human', 'csv', 'json'],
+      default: 'human',
```

Comment:
We don't need `longDescription`, `required: false`, `hidden: false`, `default: false`

Reply:
@peternhale let's drop `default:false` for booleans